src/vs/vscode.d.ts

```diff
@@ -4619,6 +4666,115 @@ declare module 'vscode' {
 		dispose(): void;
 	}
 
+	/**
+	 * Represents a terminal without a process where all interaction and output in the terminal is
+	 * controlled by an extension. This is similar to an output window but has the same VT sequence
+	 * compatibility as the regular terminal.
+	 *
+	 * Note that an instance of [Terminal](#Terminal) will be created when a TerminalRenderer is
+	 * created with all its APIs available for use by extensions. When using the Terminal object
```

Comment:
not anymore, right?

Reply:
It's still true, there's just no longer a direct link on `TerminalRenderer` to the `Terminal`.

Reply:
That would be a "normal" terminal that discoverable via `window.terminals`, `window.activeTerminal` etc?